File: Python/Condicional/Intermedio/Ejercicio_05.py
# Solicitar al usuario ingresar la cantidad de hijos y la situación de la madre
cantidad_hijos = int(input("Ingrese la cantidad de hijos: "))
madre_viuda = input("¿La madre es viuda? (Sí/No): ").lower()

# Calcular el monto del subsidio familiar
monto = 0

if cantidad_hijos <= 2:
    monto = 70
elif 3 <= cantidad_hijos <= 5:
    monto = 90
else:
    monto = 120

# Por cada hijo en edad escolar reciben S/.10.00 adicionales
hijos_edad_escolar = int(input("Ingrese la cantidad de hijos en edad escolar: "))
monto += hijos_edad_escolar * 10

# Si la madre es viuda, la familia recibe S/.20.00 adicionales
if madre_viuda == "sí":
    monto += 20

print("El monto mensual es:", monto)

File: Java/src/main/java/org/sisuns/condicionales/intermedio/Ejercicio_05.java
package org.sisuns.condicionales.intermedio;

public class Ejercicio_05 {
    public static void main(String[] args) {
        // Solicitar al usuario ingresar la cantidad de hijos y la situación de la madre
        System.out.print("Ingrese la cantidad de hijos: ");
        int cantidadHijos = 0; // Puedes asignar un valor específico si lo deseas
        // Leer la cantidad de hijos
        // cantidadHijos = ...; // Puedes utilizar Scanner si lo prefieres

        System.out.print("¿La madre es viuda? (Sí/No): ");
        String madreViuda = "No"; // Puedes asignar un valor específico si lo deseas
        // Leer la situación de la madre
        // madreViuda = ...; // Puedes utilizar Scanner si lo prefieres

        // Calcular el monto del subsidio familiar
        int monto = 0;

        if (cantidadHijos <= 2) {
            monto = 70;
        } else if (cantidadHijos >= 3 && cantidadHijos <= 5) {
            monto = 90;
        } else {
            monto = 120;
        }

        // Por cada hijo en edad escolar reciben S/.10.00 adicionales
        System.out.print("Ingrese la cantidad de hijos en edad escolar: ");
        int hijosEdadEscolar = 0; // Puedes asignar un valor específico si lo deseas
        // Leer la cantidad de hijos en edad escolar
        // hijosEdadEscolar = ...; // Puedes utilizar Scanner si lo prefieres
        monto += hijosEdadEscolar * 10;

        // Si la madre es viuda, la familia recibe S/.20.00 adicionales
        if (madreViuda.equalsIgnoreCase("Sí")) {
            monto += 20;
        }

        // Mostrar el monto mensual
        System.out.println("El monto mensual es: " + monto);
    }
}
